### PR TITLE
#765 Omitting "return" key word when returning from decorated void me…

### PR DESCRIPTION
**Pull request by gunnarmorling**
…thod

@agudian @sjaakd There you go. Feels like my first MapStruct PR for ages :)

### CODE REVIEW

File: processor/src/test/java/org/mapstruct/ap/test/decorator/PersonMapper.java [hunk 1]
```diff
@@ -33,4 +34,6 @@
     PersonDto personToPersonDto(Person person);
 
     AddressDto addressToAddressDto(Address address);
+
+    void updateAddressFromDto(AddressDto dto, @MappingTarget Address address);
```
Comment (sjaakd) [inline processor/src/test/java/org/mapstruct/ap/test/decorator/PersonMapper.java]: what about the combi: `Address updateAddressFromDto(AddressDto dto, @MappingTarget Address address);`.. Should we have a UT for that as well?

**Comment by sjaakd** [comment]
@gunnarmorling .. welcome back :smile: .. Just one comment ..

File: processor/src/test/java/org/mapstruct/ap/test/decorator/PersonMapper.java (hunk 1 above)
Comment (gunnarmorling) [inline processor/src/test/java/org/mapstruct/ap/test/decorator/PersonMapper.java]: This should be covered by `shouldDelegateNonDecoratedMethodsToDefaultImplementation()` in the same test.

**Comment by gunnarmorling** [comment]
Yeah :)

**Comment by gunnarmorling** [comment]
Rebased and applied. Thanks for reviewing, @sjaakd!

**Pull request closed without merging**